python/lsst/cp/pipe/plotPtc.py

```diff
@@ -907,27 +932,23 @@ def _plotLinearizer(self, dataset, linearizer, pdfPages):
             a2.set_xscale('linear', fontsize=labelFontSize)
             a2.set_yscale('linear', fontsize=labelFontSize)
 
-            if len(meanVecFinal):
-                pars, parsErr = linearizer.fitParams[amp], linearizer.fitParamsErr[amp]
-                k0, k0Error = pars[0], parsErr[0]
-                k1, k1Error = pars[1], parsErr[1]
-                k2, k2Error = pars[2], parsErr[2]
-                linRedChi2 = linearizer.linearityFitReducedChiSquared[amp]
-                stringLegend = (f"k0: {k0:.4}+/-{k0Error:.2e} DN\n k1: {k1:.4}+/-{k1Error:.2e} DN/t"
-                                f"\n k2: {k2:.2e}+/-{k2Error:.2e} DN/t^2 \n"
-                                r"$\chi^2_{\rm{red}}$: " + f"{linRedChi2:.4}")
-                a.scatter(timeVecFinal, meanVecFinal)
-                a.plot(timeVecFinal, funcPolynomial(pars, timeVecFinal), color='red')
-                a.text(0.03, 0.75, stringLegend, transform=a.transAxes, fontsize=legendFontSize)
-                a.set_title(f"{amp}", fontsize=titleFontSize)
-
-                linearPart = k0 + k1*timeVecFinal
-                fracLinRes = 100*(linearPart - meanVecFinal)/linearPart
-                a2.plot(meanVecFinal, fracLinRes, c='g')
-                a2.set_title(f"{amp}", fontsize=titleFontSize)
-            else:
-                a.set_title(f"{amp} (BAD)", fontsize=titleFontSize)
-                a2.set_title(f"{amp} (BAD)", fontsize=titleFontSize)
+            pars, parsErr = linearizer.fitParams[amp], linearizer.fitParamsErr[amp]
+            k0, k0Error = pars[0], parsErr[0]
+            k1, k1Error = pars[1], parsErr[1]
+            k2, k2Error = pars[2], parsErr[2]
+            linRedChi2 = linearizer.linearityFitReducedChiSquared[amp]
+            stringLegend = (f"k0: {k0:.4}+/-{k0Error:.2e} DN\n k1: {k1:.4}+/-{k1Error:.2e} DN/t"
```

Comment:
I don't see an example of this attached to the jira ticket, but doesn't this introduce spaces before the k1 and k2 lines?

Reply:
For this ticket I did not produce a linearizer; but I can check.